src/enemy.rs

```diff
@@ -30,31 +37,105 @@ Die. [V]
  */
 
 // TODO: Move to config?
-const WAVE_DURATION_SEC:u64 = 30;
-const GLOBAL_TIME_TICKER_SEC:u64 = 5;
+const WAVE_DURATION_SEC: u64 = 30;
+const GLOBAL_TIME_TICKER_SEC: u64 = 1;
 impl Plugin for EnemyPlugin {
     fn build(&self, app: &mut App) {
-        app
-            .insert_resource(CurrentWave{ num: 1, timer: Timer::new(Duration::from_secs(WAVE_DURATION_SEC ),Repeating), need_wave_spawn: true })
-            .insert_resource(GlobalTimeTickerResource(Timer::new(Duration::from_secs(GLOBAL_TIME_TICKER_SEC), TimerMode::Repeating)))
-            .add_systems(Startup, enemy_factory)
-            .add_systems(Update, (
+        app.insert_resource(CurrentWave {
+            num: 1,
+            timer: Timer::new(Duration::from_secs(WAVE_DURATION_SEC), Repeating),
+            need_wave_spawn: true,
+        })
+        .insert_resource(GlobalTimeTickerResource(Timer::new(
+            Duration::from_secs(GLOBAL_TIME_TICKER_SEC),
+            TimerMode::Repeating,
+        )))
+        .add_systems(Startup, (enemy_factory, add_enemy_count))
+        .add_systems(
+            Update,
+            (
                 spawn,
                 movement,
                 check_health,
                 change_wave,
-                global_timer_tick
-            ));
+                global_timer_tick,
+                update_enemy_count,
+                update_fps,
+            ),
+        );
+    }
+}
+
+fn add_enemy_count(mut commands: Commands) {
+    commands.spawn(
+        TextBundle::from_section(
+            "Capybaras: ",
+            TextStyle {
+                font_size: 40.0,
+                color: Color::BLACK,
+                ..default()
+            },
+        )
+        .with_style(Style {
+            position_type: PositionType::Absolute,
+            bottom: Val::Px(5.0),
+            left: Val::Px(5.0),
+            ..default()
+        }),
+    );
+
+    commands.spawn((
+        TextBundle::from_sections([
+            TextSection::new(
+                "FPS: ",
+                TextStyle {
+                    font_size: 40.0,
+                    color: Color::BLACK,
+                    ..default()
+                },
+            ),
+            TextSection::from_style(TextStyle {
+                font_size: 40.0,
+                color: Color::BLACK,
+                ..default()
+            }),
+        ]),
+        FpsText,
+    ));
+}
+
+fn update_enemy_count(
+    enemies: Query<(With<Enemy>, Without<Player>)>,
+    mut text: Query<&mut Text, Without<FpsText>>,
+) {
+    if let Ok(mut text) = text.get_single_mut() {
+        text.sections[0].value = format!("Capybaras: {}", enemies.iter().count());
+    }
+}
+
+#[derive(Component)]
+struct FpsText;
```

Comment:
why inside `enemy.rs`?

Reply:
for debugging purposes, probably should move it anywhere else

Reply:
Fixed, now it is in the `debug` module